backend/src/test/java/com/woowacourse/naepyeon/acceptance/TeamAcceptanceTest.java

```diff
@@ -411,6 +413,56 @@ void deleteNotExistTeam() {
         assertThat(response.statusCode()).isEqualTo(HttpStatus.NOT_FOUND.value());
     }
 
+    @Test
+    @DisplayName("모임에서의 내 정보를 조회한다.")
+    void findMyInfoInTeam() {
+        final String expected = "나는야모임장";
+        final MemberRegisterRequest member =
+                new MemberRegisterRequest("seungpang", "email@email.com", "12345678aA!");
+        final TokenResponseDto tokenResponseDto = 회원가입_후_로그인(member);
+        final Long teamId = 모임_생성(tokenResponseDto);
+
+        final TeamMemberResponseDto actual = 모임_가입_정보_조회(tokenResponseDto, teamId)
+                .as(TeamMemberResponseDto.class);
+        assertThat(actual.getNickname()).isEqualTo(expected);
```

Comment:
actual과 expected의 타입이 일치하는 게 좋을 것 같아요!
```suggestion
        final String actual = 모임_가입_정보_조회(tokenResponseDto, teamId)
                .as(TeamMemberResponseDto.class)
                .getNickname();
        assertThat(actual).isEqualTo(expected);
```

Reply:
반영했습니다!